doc/ref_model/chapters/chapter03.md

```diff
@@ -541,7 +541,7 @@ SyncE architecture minimally requires replacement of the internal clock of the E
 
 <a name="3.6"></a>
 ## 3.6 Storage
-The general function of storage subsystem is to provide the needed data store to various virtual and physical resources required for the delivery of a network service. In cloud infrastructure such storage may manifest itself in various ways like storage endpoints being exposed over network from software defined storage dedicated clusters or hyperconverged nodes (combining storage and other functions like compute or networking).
+The general function of storage subsystem is to provide the persistent data store for virtual resources and to the virtual and physical resources required for the delivery of a network service. Storage can be classified based on its: cost/performance, consumption model (platform native, network shared, object or archival) and capacity. In cloud infrastructure such storage may manifest itself in various ways like storage endpoints being exposed over network from software defined storage dedicated clusters or hyperconverged nodes (combining storage and other functions like compute or networking).
```

Comment:
Good addition, but "for virtual resources and to the virtual and physical resources" is a bit confusing due to the repetition. Maybe just "provide the persistent data store required for the delivery of a network service".

Reply:
Thank you Ulrich, I will include update (the sentence was messy). The primary purpose of doing this very small update was to become familiar with Unuket authouring process.